src/options/decorator.options.ts

```diff
@@ -8,6 +8,7 @@ export interface ArraySizeOptions {
 
 export interface DecoratorOptions {
     description?: string,
+    example?: unknown,
```

Comment:
The external interface to the DTOs is type composed of `DecoratorOptions | FooOptions` for some `Foo` that is based on the field type. Without `example` declared here, it's not legal to provide the value to the OpenAPI layer.